hydra-works.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec-rails', '~> 3.1'
   spec.add_development_dependency 'sqlite3'
-  spec.add_development_dependency 'solr_wrapper', '~> 0.4'
```

Comment:
Instead of changing the version of solr_wrapper, can we just do this? https://github.com/projecthydra/sufia/pull/1820